src/core/server.cc

```diff
@@ -149,6 +149,12 @@ InferenceServer::Init()
     status = CommonErrorToStatus(triton::common::AsyncWorkQueue::Initialize(
         buffer_manager_thread_count_));
   }
+
+  bool ignore_resources_and_priority =
+      (rate_limit_mode_ == RateLimitMode::RL_OFF);
```

Comment:
Default to RL_EXEC_COUNT

Reply:
Default mode is indeed RL_EXEC_COUNT.  This is done just to ignore resources and priority settings in case RateLimitMode is OFF. I will move this logic within rate limiter implementation to avoid confusion.

Reply:
Actually, that mismatches with how we handle model_control_mode_. I will leave the operation as is.